assets/js/components/KeyMetrics/FullScreenMetricSelectionApp.stories.js

```diff
@@ -21,13 +21,13 @@
  */
 import {
 	createTestRegistry,
-	provideKeyMetricsUserInputSettings,
 	provideSiteInfo,
 	WithTestRegistry,
 } from '../../../../tests/js/utils';
 import { VIEW_CONTEXT_METRIC_SELECTION } from '../../googlesitekit/constants';
 import { Provider as ViewContextProvider } from '../Root/ViewContextContext';
 import FullScreenMetricsSelectionApp from './FullScreenMetricSelectionApp';
+import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
```

Comment:
Let's move/group this constant import above together with another constant import, by putting it above the import of the `VIEW_CONTEXT_METRIC_SELECTION `

Reply:
Thank you, @zutigrm , this has been refactored in my recent commits.